packages/react-native/Libraries/Components/TextInput/InputAccessoryView.js

```diff
@@ -86,7 +87,22 @@ type Props = $ReadOnly<{|
 |}>;
 
 class InputAccessoryView extends React.Component<Props> {
+  // TODO: once the 'visionos' is implemented as a platform, we can remove this
+  componentDidMount() {
+    if (Platform.isVisionOS) {
+      warnOnce(
+        'component-unavailable',
```

Comment:
@TMisiukiewicz @okwasniewski Shouldn't the key here be something more identifiable, like `InputAccessoryView-unavailable`? 

`component-unavailable` seems a bit generic, and if it is used in other files, I believe one warning might override the other, causing some to be missed?

Reply:
That's valid point I think we should rename it

Reply:
no problem, i'll take care of it